<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogManagerUi</class>
 <widget class="QDialog" name="LogManagerUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logfile Manager</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QSplitter" name="splitter1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layout3">
      <layout class="QVBoxLayout">
       <item>
        <widget class="Q3GroupBox" name="groupBox1">
         <property name="title">
          <string>Select Bank</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="textLabel2">
            <property name="text">
             <string>Bank</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="bankSelector">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Q3GroupBox" name="groupBox2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Select Logfile</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="Q3ListView" name="fileList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="clickable">
              <bool>true</bool>
             </property>
             <property name="resizable">
              <bool>true</bool>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layout4">
      <layout class="QVBoxLayout">
       <item>
        <widget class="Q3GroupBox" name="groupBox3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Anonymized Logfile</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="Q3TextBrowser" name="fileView">
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="linkUnderline">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <enum>Q3TextEdit::FixedColumnWidth</enum>
            </property>
            <property name="wrapPolicy">
             <enum>Q3TextEdit::Anywhere</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="textLabel1">
              <property name="text">
               <string>Trustlevel</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="trustSelector">
              <item>
               <property name="text">
                <string>Untrusted</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Low Trust</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Medium Trust</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>High Trust</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Full Trust</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="spacer1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>240</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="text">
               <string>Save as...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <spacer name="spacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>181</width>
             <height>31</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>Q3TextEdit</extends>
   <header>Qt3Support/Q3TextBrowser</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>LogManagerUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
